akka-docs/src/main/paradox/scala/guide/quickstart.md

```diff
@@ -1,136 +1,26 @@
 # Quickstart
```

Comment:
When reading *The Book* (the Getting Started Guide) here in the docs this page still makes sense. It's not much content that will be duplicated, since both are delegating to the other quickstart guide.